src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -448,6 +448,16 @@ InterpreterSelectQuery::InterpreterSelectQuery(
         }
     }
 
+    /// FIXME: Memory bound aggregation may cause another reading algorithm to be used on remote replicas
+    if (settings.allow_experimental_parallel_reading_from_replicas && settings.enable_memory_bound_merging_of_aggregation_results)
+        context->setSetting("enable_memory_bound_merging_of_aggregation_results", false);
```

Comment:
shouldn't the same comment apply as for the Joins? you're disabling memory_bound_merging for all queries if `allow_experimental_parallel_reading_from_replicas` is enabled.

Reply:
Discussed in DM, this is a temporary solution, because I faced with a bug which will be resolved with https://github.com/ClickHouse/ClickHouse/pull/45892